src/py/hello-templated.py

```diff
@@ -0,0 +1,10 @@
+# this Python snippet is stored as src/py/hello-templated.py
```

Comment:
I am not sure if these `Hello World` examples are needed. Readers maybe should follow another tutorial if needed.

Reply:
OK will try to introduce template using NewtonRaphson directly.

Reply:
Found even better solution by using f-strings instead of templates.